tutorials/Getting_Started/README.md

```diff
@@ -66,7 +64,7 @@ pluginManagement {
     }
 }
 ```
-Then create `build.gradle.kts` with the following content:
+3. Then create a kotlin script file `build.gradle.kts` with the following content:
```

Comment:
3. Then create a Kotlin script file `build.gradle.kts` with the following content: